terraform/Makefile

```diff
@@ -35,4 +39,12 @@ all:
 	make build
 	make plan
 	make apply
-.PHONY: all
\ No newline at end of file
+.PHONY: all
+
+CI:
```

Comment:
Will CI: run only after the first environment creation? If so, it's ok as make build is not being run here otherwise it should be run.

Reply:
CI will be run by the GH Action when new PR's merge to Master